DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/task_sequences/README.md

```diff
@@ -55,70 +55,109 @@ You can also find more information about [troubleshooting build errors](/DirectP
 
 ![](assets/simulation_task_sequence.png)
 
-Observe in the above graphic that the output pipe produces results almost immediately.
 
-### Simple Code Example 
+### Code Example 
 
 To use a task sequence in your design, include the `<sycl/ext/intel/experimental/task_sequence.hpp>` header file in your source code. The `task_sequence` class is a templated class with 3 parameters:
 
 | Template Parameter | Type     | Default Value | Description
 |--------------------|----------|---------------|---
 | Task function      | callable | N/A           | A callable object `f` that defines the asynchronous task to be associated with the `task_sequence`. The callable object `f` must meet the following requirements: <br> • The object `f` must be statically resolvable at compile time, which means it is not a function pointer. <br> • The object `f` must not be an overloaded function. <br> • The return type (`ReturnT`) and argument types (`ArgsT…`) of object f must be resolvable and fixed.
-| Invocation Capacity* | `uint32_t` | 1 | The size of the hardware queue instantiated for `async()` function calls. This parameter value corresponds to the minimum number of outstanding `async()` function calls to be supported. When the outstanding number of `async()` function calls reach this value, further calls may block until the number of outstanding calls is reduced to the `invocation_capacity`. The default value of this parameter is 1.
-| Response Capacity* | `uint32_t` | 1 | The size of the hardware queue instantiated to hold task function results. This parameter value corresponds to the maximum number of outstanding `async()` calls such that all outstanding tasks are guaranteed to make forward progress. Further `async()` calls may block until the number of outstanding calls reduce to the `response_capacity`. The default value of this parameter is 1.
+| Invocation Capacity* | `uint32_t` | 1 | The size of the hardware queue instantiated for `async()` function calls. This parameter value corresponds to the minimum number of outstanding `async()` function calls to be supported. When the outstanding number of `async()` function calls reaches this value, further calls may block until the number of outstanding calls is reduced to the `invocation_capacity`. The default value of this parameter is 1.
+| Response Capacity* | `uint32_t` | 1 | The size of the hardware queue instantiated to hold task function results. This parameter value corresponds to the maximum number of outstanding `async()` calls such that all outstanding tasks are guaranteed to make forward progress. Further `async()` calls may block until the number of outstanding calls reduces to the `response_capacity`. The default value of this parameter is 1.
+*Invocation capacity and response capacity are optional 
```

Comment:
```suggestion

*Invocation capacity and response capacity are optional 
```
add one or two newlines so that this doesn't get included in the table

![image](https://github.com/chanjunc-intel/oneAPI-samples/assets/86071037/b6b1b740-c5e0-401b-9d24-40dc1ec16989)